src/utils/MavenUtil.ts

```diff
@@ -165,3 +189,23 @@ export function findChildMavenModules(xmlString: string): Map<string, string[]>
     });
     return childrenMap;
 }
+
+/**
+ * Return true if the liberty-maven-plugin version is compatible
+ * for dev mode with containers 
+ * 
+ * @param plugin JS object for liberty-maven-plugin
+ */
+function mavenPluginVersionValid(plugin: any): boolean {
+    if (plugin.version === undefined) {
+        return true;
+    }
+    if (plugin.version[0] !== undefined) {
+        // grab the first 2 digits, ie. if version is 3.3.1 return a float of `3.3`
+        let versionStart = plugin.version[0].substring(0,3);
```

Comment:
If at some point the version components reach 2 digits, this is going to have problems.  Although at that point we can probably remove this check anyways.

Same applies to Gradle.

Reply:
Left as-is for now

Reply:
FYI: the semver package might be useful for this https://www.npmjs.com/package/semver

Reply:
Updated to use semver - thanks Ryan!